### PR TITLE
Indicate When Project is Under Beta Review

### LINKED ISSUE #3404: When a project transitions to beta approved but not launch approved the not a Zooniverse project disclaiminer disappears and then it looks like it is launch approved
In regards to this disclaimer message on the bottom of non-Zooniverse approved projects.  'This project has been built using the Zooniverse Project Builder but is not yet an official Zooniverse project. Queries and issues relating to this project directed at the Zooniverse Team may not receive any response.' 

A recent change seems to now make this message go away when approved for beta test.  So the project is in beta test approved, but not launched approved. The only difference then separating Zooniverse approved and beta is that Zooniverse-approved appear on the projects page. I think it would be good to have the disclaimer message show on beta-approved but not launch approved projects again

### LINKED ISSUE #3481: Add more visual indication that a project is still in review
( See attached image )

**Avatar**
  Border: 3px #00979D;

**Label**
  font-family : Open Sans;
  font-weight : 300;
  font-size : 8px;
  letter-spacing : 0.25em;
  color : #00979D;

screen shot 2017-02-14 at 10 17 15 am

### PR DESCRIPTION
**Pull request by wgranger**
Fixes #3404 
Fixes #3481 

Describe your changes.
Three main changes:

- Visual indicator on main project button when under beta review

- Border around page when project is under beta review

- Add the "not an official Zooniverse project" disclaimer for projects under beta review

# Review Checklist

- [x] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [x] Does it work on mobile?
- [x] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [x] Did you deploy a staging branch?
- [ ] Did you get any type checking errors from [Babel Typecheck](https://github.com/codemix/babel-plugin-typecheck)

https://beta-review.pfe-preview.zooniverse.org/

## Optional

- [ ] If it's a new component, is it in ES6? Is it clear of warnings from ESLint?
- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?

### CODE REVIEW

File: app/pages/project/index.cjsx [hunk 1]
```diff
@@ -228,7 +228,17 @@ ProjectPage = React.createClass
     getUserRoles.then (userRoles) =>
       isAdmin() or 'owner' in userRoles or 'collaborator' in userRoles
 
+  renderProjectName: (betaApproved) ->
+    if betaApproved
+      <span>
+        <p>Under Review</p>
```
Comment (eatyourgreens) [inline app/pages/project/index.cjsx]: `p` isn't allowed as a child of `span`, so this is invalid HTML.

File: app/pages/project/index.cjsx [hunk 2]
```diff
@@ -261,13 +271,13 @@ ProjectPage = React.createClass
             Visit {@props.project.display_name}
           </a>
         else
-          <IndexLink to="#{projectPath}" activeClassName="active" className="tabbed-content-tab" onClick={logClick?.bind this, 'project.nav.home'}>
+          <IndexLink to="#{projectPath}" activeClassName="active" className="tabbed-content-tab #{('beta-approved' if betaApproved) ? ''}" onClick={logClick?.bind this, 'project.nav.home'}>
```
Comment (eatyourgreens) [inline app/pages/project/index.cjsx]: PFE uses the `classnames` module, which can make logic like this a bit easier to deal with. Have a look at https://www.npmjs.com/package/classnames

File: app/pages/project/index.cjsx [hunk 3]
```diff
@@ -464,8 +474,13 @@ ProjectPageController = React.createClass
 
   render: ->
     slug = @props.params.owner + '/' + @props.params.name
+    betaApproved = @state.project?.beta_approved
 
     <div className="project-page-wrapper">
+
+      {if betaApproved
+        <div className="beta-border"></div>}
```
Comment (eatyourgreens) [inline app/pages/project/index.cjsx]: Maybe just add this class to the wrapper? Or to the background image `div`?

**Comment by wgranger** [comment]
This project should still be under beta review: https://beta-review.pfe-preview.zooniverse.org/projects/anabellecardoso/elephant-expedition/?env=production

**Comment by wgranger** [comment]
Oh wait a minute. There might be an issue with the styling on that latest commit.

File: css/tabbed-content.styl [hunk 4]
```diff
@@ -42,7 +42,7 @@
       font-size: 0.8em
       margin: 0
 
-    span
+    div
```
Comment (eatyourgreens) [inline css/tabbed-content.styl]: `div, span` here to preserve styles on the other navigation links?

File: app/pages/project/index.cjsx (hunk 3 above)
Comment (wgranger) [inline app/pages/project/index.cjsx]: The `.beta-border` class doesn't play well with the outer `div` and restricts scrolling, so I added it back as a separate `div`.

**Comment by eatyourgreens** [comment]
Looks good to me. Does the `pointer-events` style interfere with drawing tasks?

**Comment by wgranger** [comment]
Ooo. Good call. It shouldn't, since it behaves the same way as the Admin Indicator construction-ish border, but I can double check.

**Comment by wgranger** [comment]
It looks good. Just checked on a drawing project.

**Comment by eatyourgreens** [comment]
👍

**Pull request merged**